freqtrade/leverage/liquidation_price.py

```diff
@@ -41,147 +53,90 @@ def exception(
+        f"{exchange} does not support {collateral.value} Mode {trading_mode.value} trading ")
 
 
 def binance(
     open_rate: float,
     is_short: bool,
     leverage: float,
-    margin_mode: MarginMode,
     trading_mode: TradingMode,
     collateral: Collateral,
-    **kwargs
+    wallet_balance: float,
+    maintenance_margin_ex_1: float,
+    unrealized_pnl_ex_1: float,
+    maintenance_amount: float,
+    position_1: float,
+    entry_price_1: float,
+    maintenance_margin_rate: float,
 ):
     r"""
         Calculates the liquidation price on Binance
         :param open_rate: open_rate
         :param is_short: true or false
         :param leverage: leverage in float
-        :param margin_mode: one-way or hedge
         :param trading_mode: spot, margin, futures
         :param collateral: cross, isolated
 
-        :param \**kwargs:
-            See below
-
-        :Keyword Arguments:
-            * *wallet_balance* (``float``) --
-              Wallet Balance is crossWalletBalance in Cross-Margin Mode
-              Wallet Balance is isolatedWalletBalance in Isolated Margin Mode
-
-            * *maintenance_margin_ex_1* (``float``) --
-              Maintenance Margin of all other contracts, excluding Contract 1.
-              If it is an isolated margin mode, then TMM=0
-
-            * *unrealized_pnl_ex_1* (``float``) --
-              Unrealized PNL of all other contracts, excluding Contract 1.
-              If it is an isolated margin mode, then UPNL=0
-
-            * *maintenance_amount_both* (``float``) --
-              Maintenance Amount of BOTH position (one-way mode)
-
-            * *maintenance_amount_long* (``float``) --
-              Maintenance Amount of LONG position (hedge mode)
+        :param wallet_balance: Wallet Balance is crossWalletBalance in Cross-Margin Mode.
+          Wallet Balance is isolatedWalletBalance in Isolated Margin Mode
 
-            * *maintenance_amount_short* (``float``) --
-              Maintenance Amount of SHORT position (hedge mode)
+        :param maintenance_margin_ex_1: Maintenance Margin of all other contracts, excluding Contract 1.
+          If it is an isolated margin mode, then TMM=0
 
-            * *side_1_both* (``int``) --
-              Direction of BOTH position, 1 as long position, -1 as short position
-              Derived from is_short
+        :param unrealized_pnl_ex_1: Unrealized PNL of all other contracts, excluding Contract 1.
+          If it is an isolated margin mode, then UPNL=0
 
-            * *position_1_both* (``float``) --
-              Absolute value of BOTH position size (one-way mode)
+        :param maintenance_amount: Maintenance Amount of position (one-way mode)
 
-            * *entry_price_1_both* (``float``) --
-              Entry Price of BOTH position (one-way mode)
+        :param position_1: Absolute value of position size (one-way mode)
```

Comment:
can you take the 1 off this and entry_price ? when we include hedge mode, I think it might be better to just use a tuple or an array instead of having 3 separate parameters for the same thing

Reply:
https://github.com/freqtrade/freqtrade/pull/5576/commits/da00b21cd50c1341dfd97b67e72257378d5ada74